docs/networking/rocm.rst

```diff
@@ -0,0 +1,119 @@
+ROCm
+====
+
+How do I build Open MPI with ROCm-aware support?
+------------------------------------------------
+
+ROCm-aware support means that the MPI library can send and receive GPU
+buffers directly. As of today, ROCm support is available through
+UCX. While other communication transports might work as well, UCX is
+the only transport formally supported in Open MPI for ROCm devices in
+the current version.
```

Comment:
You can use `|ompi_ver|` to list Open MPI's full version (the `|foo|` notation indicates a macro; we defined the `ompi_ver` macro to be something like "v5.0.0" or "v5.0.1" or the like.  You can also use `|ompi_series|` for values like "v5.0.x" or "v4.1.x", or whatever is appropriate on this branch.